examples/dp_scaffold_example/server.py

```diff
@@ -12,13 +13,15 @@
 from fl4health.server.scaffold_server import DPScaffoldServer
 from fl4health.strategies.scaffold import Scaffold
 from fl4health.utils.config import load_config
+from fl4health.utils.functions import privacy_validate_and_fix_modules
 from fl4health.utils.metric_aggregation import evaluate_metrics_aggregation_fn, fit_metrics_aggregation_fn
 
 
 def get_initial_model_information() -> Tuple[Parameters, Parameters]:
     # Initializing the model parameters on the server side.
     # Currently uses the Pytorch default initialization for the model parameters.
-    initial_model = MnistNet()
+    initial_model: nn.Module = MnistNet()
+    initial_model, _ = privacy_validate_and_fix_modules(initial_model)
```

Comment:
Note: This model wasn't causing any problems, but I wanted the examples to reflect a situation where the server should pass the appropriate model parameters **after** validation and correction to ensure that all the model weights line up properly